src/state.rs

```diff
@@ -26,6 +26,13 @@ pub mod persist;
 #[derive(Clone, Default, Canon)]
 pub struct Contracts(Map<ContractId, Contract>);
 
+use once_cell::sync::Lazy;
+use parking_lot::RwLock;
+use std::sync::Arc;
+
+pub(crate) static HOST_MODULES: Lazy<Arc<RwLock<HostModules>>> =
+    Lazy::new(|| Arc::new(RwLock::new(HostModules::default())));
+
```

Comment:
It was described in the commit message: https://github.com/dusk-network/rusk-vm/pull/298/commits/3f52c141efef9e971b5e336d1a807b1ea920cc27

I usually avoid to describe "changes" in the code since you don't have a "before / after" (where in the commit you have), so it's a bit hard to formulate "Why", but I can write part of the commit message as comment here if you think is needed:
```
/// A thread-safe singleton used to register host modules for Rusk VM instances
```
Or something like that.